README.md

```diff
@@ -47,6 +47,7 @@ script files.
 -- Very simple variables can be used. You can reference assets within a script which can be used
 -- to define who is speaking a line of dialogue, or if they implement the ISupertalkDisplayInterface
 -- they can be embedded within lines of dialogue themselves.
+-- Supertalk will attempt to resolve the variable to an asset as long as the right-hand side of the assignment contains a forward slash or two backslashes.
```

Comment:
*as long as the value starts with a slash. Containing a slash otherwise shouldn't result in an asset reference, and if it does that's a bug that I need to look into.